test/extended/prometheus/prometheus.go

```diff
@@ -292,19 +293,25 @@ func waitForServiceAccountInNamespace(c clientset.Interface, ns, serviceAccountN
 	return err
 }
 
+var (
+	processTemplate bool
+	mux             sync.Mutex
+)
+
 func bringUpPrometheusFromTemplate(oc *exutil.CLI) (ns, host, bearerToken string, statsPort int) {
 	ns = oc.KubeFramework().Namespace.Name
 	host = "prometheus.kube-system.svc"
 	statsPort = 443
-	mustCreate := false
+	mux.Lock()
```

Comment:
Won't work, other jobs are run in separate processes.

Reply:
Ah ... gotcha.

So seems like we need to interpret the error, and if it is "already exists", do not abort the test case.

I'll circle back and give that a go (unless some other direction is supplied).

Reply:
I'll see if I can remove the lgtm label

/hold

Reply:
appears I can :-)